pkg/signup/service/signup_service_test.go

```diff
@@ -1241,6 +1246,28 @@ func (s *TestSignupServiceSuite) TestGetSignupByUsernameOK() {
 	err = s.FakeMasterUserRecordClient.Tracker.Add(mur)
 	require.NoError(s.T(), err)
 
+	inf := fake.NewFakeInformer()
+	inf.GetUserSignupFunc = func(name string) (*toolchainv1alpha1.UserSignup, error) {
+		if name == us.Name {
+			return us, nil
+		}
+		return nil, apierrors.NewNotFound(schema.GroupResource{}, name)
+	}
+	inf.GetMurFunc = func(name string) (*toolchainv1alpha1.MasterUserRecord, error) {
+		if name == mur.Name {
+			return mur, nil
+		}
+		return nil, apierrors.NewNotFound(schema.GroupResource{}, name)
+	}
+	s.Application.MockInformerService(inf)
```

Comment:
I don't think that we need anything from the fake informers to be set up
```suggestion
```

Reply:
You're right, looks like this was unnecessary.